app/models/card.rb

```diff
@@ -1,2 +1,16 @@
 class Card < ApplicationRecord
+  validates :original_text, :translated_text, :review_date, presence: true
+  validate :original_differs_from_trans
+  after_initialize :set_review_date, on: :create
+
+  private 
+  def original_differs_from_trans
+    if original_text.downcase == translated_text.downcase
+      errors.add(:translated_text, "translated text can't be equal with original text")
+    end
+  end
+
+  def set_review_date
+    self.review_date = Time.now + 3.days
```

Comment:
are you sure what you need self. ?

Reply:
Without self it does not work, when i open new form of the card.
As i understand, without 'self' there create's new variable in local scope, instead of setting value of class attr